### PR TITLE
Test for RNASeq DSS and RNASeq adapters bugfix

### PR DESCRIPTION
**Pull request by mzganec**
The `id_attribute` input was added to RNA-seq DSS to set the corresponding input of the underlying process. The input schema remains intact for user friendliness. The DSS will soon be moved to another repository.

### CODE REVIEW

**Comment by mstajdohar** [comment]
@dblenkus Please review tests.

File: resolwe_bio/tests/workflows/test_rna_seq.py [hunk 1]
```diff
@@ -137,3 +143,53 @@ def test_rnaseq_paired_workflow(self):
...
+
+class RNASeqDSSTestCase(TransactionBioProcessTestCase, LiveServerTestCase):
+    @with_resolwe_host
+    def test_rnaseq_dss(self):
+        single_reads = self.prepare_reads()
+        paired_reads = self.prepare_paired_reads()
+        genome = self.prepare_genome()
+        genome.slug = 'genome-mm10'
+        genome.save()
+        assign_perm('view_data', AnonymousUser(), genome)
```
Comment (dblenkus) [inline resolwe_bio/tests/workflows/test_rna_seq.py]: Add new lines between sections, i.e. `reads`, `genome`, `annotation`, `adapters`

File: resolwe_bio/tests/workflows/test_rna_seq.py [hunk 2]
```diff
@@ -137,3 +143,53 @@ def test_rnaseq_paired_workflow(self):
...
+
+        workflow = Data.objects.last()
+        self.assertFile(workflow, 'rc', 'workflow_rnaseq_single_rc.tab.gz', compression='gzip')
+        self.assertFields(workflow, 'exp_type', 'TPM')
+        self.assertFields(workflow, 'source', 'DICTYBASE')
+
+        self.run_process('dss-rna-seq', {
+            'genome_and_annotation': 'mm',
+            'reads': paired_reads.id,
+            'trailing': 1,
```
Comment (dblenkus) [inline resolwe_bio/tests/workflows/test_rna_seq.py]: What is the `trailing` parameter?

File: resolwe_bio/tests/workflows/test_rna_seq.py [hunk 3]
```diff
@@ -137,3 +143,53 @@ def test_rnaseq_paired_workflow(self):
...
+
+        for data in Data.objects.all():
+            self.assertStatus(data, Data.STATUS_DONE)
+
+        workflow = Data.objects.last()
+        self.assertFile(workflow, 'rc', 'workflow_rnaseq_single_rc.tab.gz', compression='gzip')
+        self.assertFields(workflow, 'exp_type', 'TPM')
+        self.assertFields(workflow, 'source', 'DICTYBASE')
+
+        self.run_process('dss-rna-seq', {
```
Comment (dblenkus) [inline resolwe_bio/tests/workflows/test_rna_seq.py]: Add a comment above the `run_process` call about what are you testing.

**Pull request merged**